libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -355,6 +355,21 @@ AwsIotProvisioningError_t _AwsIotProvisioning_SerializeCreateKeysAndCertificateR
 
 /*------------------------------------------------------------------*/
 
+bool _AwsIotProvisioning_SerializeCreateCertificateFromCsrRequestPayload( const char * pCertificateSigningRequest,
```

Comment:
This function just returns true ? Is serialization code going to get added later ?

Reply:
Yes, stubbed out to reduce the scope of this PR.